src/app/shared/services/documentStore/document-store-service.service.spec.ts

```diff
@@ -1,15 +1,38 @@
 import { TestBed, inject } from '@angular/core/testing';
-
+import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
+import { ConfigService } from '../../../config.service';
+import { TransferState } from '@angular/platform-browser';
```

Comment:
Just curious, I thought @smogese removed angular universal? In which case we should probably remove all the transfer state from the frontend code too? Just redundant otherwise

Reply:
The site should not use angular universal going forward. I could not strip it out completely as the PDF annotations use it for something.

In addition, the upload service needs to be removed as it was decided that this functionality would not be provided by JUI and the backend code for it has recently been removed. Forgot there was an FE service.